tests/test_biosamples_submission.py

```diff
@@ -433,7 +440,7 @@ def test_convert_json_to_bsd_json(self):
     def test_convert_old_json_to_bsd_json(self):
         # JSON as generated by eva-sub-cli xlsx2json prior to v0.4.13
         bio_sample_object = {
-            "name": "LH1",
+            "bioSampleName": "LH1",
```

Comment:
I swear I saw you changed this:
```suggestion
            "name": "LH1",
```